packages/pages/src/vite-plugin/modules/plugin.ts

```diff
@@ -0,0 +1,173 @@
+import { build, createLogger } from "vite";
+import { ProjectStructure } from "../../common/src/project/structure.js";
+import { glob } from "glob";
+import path from "node:path";
+import fs from "node:fs";
+import { convertToPosixPath } from "../../common/src/template/paths.js";
+import { processEnvVariables } from "../../util/processEnvVariables.js";
+import { nodePolyfills } from "vite-plugin-node-polyfills";
+import pc from "picocolors";
+import { addResponseHeadersToConfigYaml } from "../../util/editConfigYaml.js";
+import SourceFileParser, {
+  createTsMorphProject,
+} from "../../common/src/parsers/sourceFileParser.js";
+
+const wrappedCode = (moduleName: string, containerName: string): string => {
+  return `\nconst moduleContainerForBuildUseOnly = document.getElementById('${containerName}');
```

Comment:
This would be easier to read if formatted using triple backticks.

Reply:
I don't think that is possible in this code. I get 
```
This expression is not callable.
Type 'String' has no call signatures.
```

Reply:
Woops, triple backticks is Go 😅.